docs/source/api_reference/featurizers.rst

```diff
@@ -472,6 +472,27 @@ DummyFeaturizer
   :members:
   :inherited-members:
 
```

Comment:
Rather than have these with "other featurizers", let's add a new section in the docs for tokenizers

Reply:
added new section in docs